### ISSUE TITLE
Escolha dos atributos

### ISSUE THREAD

**Issue by fparreirinha** [fparreirinha/Trabalho-Python#4]
A escolha dos atributos nos 3 classisficadores precisam ser os mesmos, para comparações posteriores.

**Comment by fparreirinha** [comment]
Os 3 classificadores foram treinados com as mesmas features, acredito que tenha surgido a dúvida pelo classificador de regressão de árvore de decisão, onde realizamos o primeiro treino com apenas uma feature, depois o treinamos com as mesmas features dos outros classificadores (['horsepower','engine_size','curb_weight','highway_mpg','peak_rpm']), justamente para comparação, mas também achamos interessante, utilizarmos a função feature_importances_, que traz a importância de uma feature (normalizada) pela redução total do erro trazido por esta feature, e então realizamos o terceiro treino.

fonte:
http://ogrisel.github.io/scikit-learn.org/sklearn-tutorial/modules/generated/sklearn.tree.DecisionTreeRegressor.html
